libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/puck/LocationPuckOptions.kt

```diff
@@ -15,11 +17,12 @@ import com.mapbox.navigation.ui.maps.R
  * `defaultPuck()` on the [Builder] and that would be used for pucks in all navigation states.
  *
  * @param context from which to reference puck drawables
- * @param freeDrivePuck stores the puck appearance to be displayed in free drive state
- * @param destinationPreviewPuck stores the puck appearance to be displayed in destination preview state
- * @param routePreviewPuck stores the puck appearance to be displayed in route preview state
- * @param activeNavigationPuck stores the puck appearance to be displayed in active navigation state
- * @param arrivalPuck stores the puck appearance to be displayed in arrival state
+ * @param freeDrivePuck stores the puck appearance to be displayed in free drive state when in the [NavigationCameraState.FOLLOWING]
+ * @param destinationPreviewPuck stores the puck appearance to be displayed in destination preview state when in the [NavigationCameraState.FOLLOWING]
+ * @param routePreviewPuck stores the puck appearance to be displayed in route preview state when in the [NavigationCameraState.FOLLOWING]
+ * @param activeNavigationPuck stores the puck appearance to be displayed in active navigation state when in the [NavigationCameraState.FOLLOWING]
+ * @param arrivalPuck stores the puck appearance to be displayed in arrival state when in the [NavigationCameraState.FOLLOWING]
+ * @param idlePuck stores the puck appearance to be displayed for all navigation states when in the [NavigationCameraState.IDLE] or the [NavigationCameraState.OVERVIEW]
```

Comment:
Wondering if API can be designed in a way that makes these limitations clear without reading the docs. 
Also one may wonder why the puck can be customized per navigation state with the FOLLOWING camera, but not with other cameras. 
Approving because I don't see how this can be improved. The main problem seems to be the fact that we can't break existing `LocationPuckOptions` functionality.

Reply:
Yeah. I think it's best to wait for some customer feedback on that one.